addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -307,7 +307,9 @@ body.editor_enable.editor_has_snippets {
                     text-align: center;
 
                     > we-customizeblock-option {
-                        display: inline-block; // Ideally -> display: contents;
+                        &, .fa, .fas, .fab, .far {
+                            display: inline-block !important; // Ideally -> display: contents;
```

Comment:
Remove the comment (I put it but it is stupid). Also FA5 is not there yet and never will in 13.2... can you remove the extra rules?

Reply:
It's done